server/config.ts

```diff
@@ -130,7 +130,7 @@ export default {
         response: Number(get('PRISONER_SEARCH_API_TIMEOUT_RESPONSE', 10000)),
```

Comment:
```suggestion
        response: Number(get('PRISONER_SEARCH_API_TIMEOUT_RESPONSE', 20000)),
```
do you also need to increase this one?